block/manager.go

```diff
@@ -826,38 +826,49 @@ func (m *Manager) recordMetrics(block *types.Block) {
 	m.metrics.CommittedHeight.Set(float64(block.Height()))
 }
 
-func (m *Manager) submitBlocksToDA(ctx context.Context) error {
-	submitted := false
+func (m *Manager) submitBlocksToDA(ctx context.Context) (uint64, error) {
+	submittedAll := false
```

Comment:
nitpicking: Maybe just `finished` or `done`?

Reply:
changed to `submittedAllBlocks`